_rules/descriptive-labels-f0c5c5.md

```diff
@@ -0,0 +1,280 @@
+---
+id: f0c5c5
+name: Form field label is descriptive
+rule_type: atomic
+description: |
+  This rule checks that labels describe the purpose of form field elements.
+accessibility_requirements: 
+  wcag21:2.4.6: # Headings and labels (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+- DOM Tree
+- CSS Styling
+authors:
+- Anne Thyme Nørregaard
+- Dagfinn Rømen
+- Geir Sindre Fossøy
+---
+
+## Applicability
```

Comment:
I guess this is because this rule is only concerned with form fields. This is a subpart of the SC, but this kind of situation (rule only addressing one small part of the SC) is rather common. We'll need more rules to address the full SC…

Reply:
Also, I think this is due to the change from `aria-labelledby` to "accessible name". I think it was quite easy to target "the `label` or `aria-labelledby` element" but it is somewhat difficult to target "the accessible name", hence moving the target to the form field in order to be able to make the rule broader (including `aria-label`, …)